core/domain/stats_services.py

```diff
@@ -169,6 +169,17 @@ def get_versions_for_exploration_stats(exploration_id):
         exploration_id)
 
 
+def get_total_unresolved_answers_for_exploration(exploration_id):
+    """Returns the sum total of unresolved answers (across all states) of an
+    exploration."""
+    exploration = exp_services.get_exploration_by_id(exploration_id)
+    state_names = exploration.states.keys()
+
+    return sum(stats_domain.StateRuleAnswerLog.get(
```

Comment:
Is this slow? If you're going to end up doing it for multiple explorations at a time, you may need to batch things. Otherwise you'll be doing N get() calls in sequence rather than 1 get_multi(); the former is much slower.

/cc @BenHenning as this intersects with his answer-migration work